ts/src/okx.ts

```diff
@@ -553,7 +553,7 @@ export default class okx extends Exchange {
                     '51027': BadSymbol, // Contract expired
                     '51028': BadSymbol, // Contract under delivery
                     '51029': BadSymbol, // Contract is being settled
-                    '51030': BadSymbol, // Funding fee is being settled
+                    '51030': ContractSettled, // Funding fee is being settled
```

Comment:
Can you point me to this error in the docs? Second, aren't the errors above related ?

Reply:
> Can you point me to this error in the docs? 

I can't provide a direct link, but all the errors are visible under https://www.okx.com/docs-v5/en/#error-code-rest-api-public and you can just cmd/cntl + f for 51030

> aren't the errors above related ?

Yes they would be, I didn't notice those ones

Reply:
Maybe a different name would work better, so that it can include these as well

                    '51021': BadSymbol, // Contract to be listed
                    '51022': BadSymbol, // Contract suspended
                    '51027': BadSymbol, // Contract expired
                    '51028': BadSymbol, // Contract under delivery
                    '51029': BadSymbol, // Contract is being settled
                    '51030': ContractSettled, // Funding fee is being settled

Reply:
> Maybe a different name would work better, so that it can include these as well

@samgermain yeah maybe, do you have any suggestions for the name?

Reply:
I cannot think of one